mlc_config.json

```
@@ -0,0 +1,10 @@
+{
+  "ignorePatterns": [
+    {
+      "pattern": "^https://github.com/asyncapi/glee/blob/086c449/src"
+    },
+    {
+      "pattern": "^https://asyncapi.org/glee/docs"
```

I would follow the same pattern as the rest of the tools:

```suggestion
      "pattern": "^https://www.asyncapi.com/tools/glee"
```

However, I'm not sure if we should even have that yet. We can add it later.

---

where do you think https://github.com/asyncapi/glee/blob/master/examples/social-network/notifications-service/README.md?plain=1#L15 should point to for now? since the current link is dead. 🤔

---

Oh! I didn't notice this one. There are no docs yet so I'd point it to this repo:
```suggestion
      "pattern": "^https://www.github.com/asyncapi/glee"
```